cmake/developer_package/download/dependency_solver.cmake

```diff
@@ -176,9 +176,9 @@ function(reset_deps_cache)
         foreach(var_name IN LISTS ARGN)
             unset(${var_name} CACHE)
         endforeach()
-        foreach(var_name IN LISTS ARGN)
-            unset(ENV{${var_name}})
-        endforeach()
+        # foreach(var_name IN LISTS ARGN)
+        #     unset(ENV{${var_name}})
+        # endforeach()
```

Comment:
@vinograd47 I've commented this code because it resets manually set TBBROOT variable.. Is it OK?

Reply:
looks like I have faced with the same issue with TEMP folder. I set TBBROOT to point to temp and that is why my env variables was unset. 

Ok, I revert this small change.

Reply:
Yes, the whole logic of the `reset_deps_cache` and related function based on assumption that packages located in TEMP folder was automatically downloaded by CMake script itself and that user custom packages are located in another place. The TEMP folder is assumed to be owned by CMake script and, for example, it might have some clean up logic for it. User shouldn't place anything there.

Reply:
Ok, thanks for the explanations!

See https://github.com/openvinotoolkit/openvino/pull/5659